server/helpers/task.js

```diff
@@ -53,57 +47,55 @@ const createTask = async data => {
     await departmentData.save()
     return { message: 'Task created successfully.' }
   } catch (err) {
-    throw Error(`Could not create task, error: ${err}`)
+    throw Error(`${err?.message ?? 'Something went wrong'}`)
   }
 }
 
 const createManyTask = async (tasks, freelancerId = null) => {
   try {
     const { getBusinessWithoutPopulate } = require('./business')
-    const { getDepartmentWithoutPopulate } = require('./department')
-    const { getContractWithoutPopulate } = require('./contract')
     const { getTagsWithoutPopulate } = require('./tags')
+    const { getDepartmentWithoutPopulate } = require('./department')
```

Comment:
can you please make these imports on top of the page to maintain code symmetry?

Reply:
When some modules are depend on each other then circular dependency issues occur, therefore I'm importing  method in this way.

Reply:
can you please fix these circular dependency issues as its gonna cause alot of issues in future.
looping-in: @Alien4Hire

Reply:
![image](https://github.com/Unzipped-Technologies/unzipped/assets/14105328/8b7affbd-c2e0-4883-913a-94bc32b54110)
![circularDependency](https://github.com/Unzipped-Technologies/unzipped/assets/14105328/f8232fd2-5842-4903-a50b-1edef0b12236)
It will take a lot of time to refactor code. For now it's better to use this method. It occur when module is also import in module B and module B is import in module A.